### DISCUSSION TITLE
How to set vlaidaterc file of input.validation field

### DISCUSSION THREAD

**Discussion by tatsuya-asami** [orval-labs/orval#1693] [Q&A]
According to [this document](https://orval.dev/reference/configuration/input), we should create `.validaterc` file to change open api validation rule.

However, `.validaterc` file might not be read if I create file `.validaterc` in the root of my repository.

I would like to get some example to modify validation rule.

I read the code of this file, but could not understand how to modify it.
https://github.com/orval-labs/orval/blob/852dce7746b33a246e408b4284a1f5f71caf4390/packages/core/src/utils/validator.ts

**Comment by melloware** [comment]
I actually didn't know this feature even existed @anymaniax ?
